Jupyter/readme.md

```diff
@@ -5,27 +5,27 @@ and [Python installation](https://github.com/QuantConnect/Lean/tree/master/Algor
 
 ### [Windows](https://github.com/QuantConnect/Lean#windows)
 **1. Install Jupyter and its dependencies:**
-   1. Install Jupyter
+   1. Install Jupyter:
 ```
     pip install jupyter
 ```
  2.  Install [QuantConnect Python API](https://pypi.python.org/pypi/quantconnect/0.1)
  ```
     pip install quantconnect
 ```
- 3.  Install pythonnet
- If you haven't yet, You should first set up [git](https://help.github.com/articles/set-up-git/). Then
+ 3.  Install PythonNet:
+ If you haven't installed Git yet you should first setup [git](https://help.github.com/articles/set-up-git/), then install PythonNet.
  ```
     pip install --egg  git+https://github.com/QuantConnect/pythonnet
```

Comment:
Are there empty spaces before the commands necessary? We don't have them in other installation instructions.